applications/rag/variables.tf

```diff
@@ -261,7 +261,7 @@ variable "private_cluster" {
 
 variable "autopilot_cluster" {
   type    = bool
-  default = false
+  default = true
```

Comment:
Strongly prefer to not do this until we resolve Ray autoscaler timeout issues. I don't think a 25% success rate for job submission is acceptable on AP clusters.

Reply:
Addressed in the diffbase: http://go/ai-on-gke/pull/361